src/screens/UserManagement/UserRevamp/UserUtils.res

```diff
@@ -93,10 +93,15 @@ let modulesWithUserAccess = (
       let accessGroup = userAccessGroup->Array.find(group => group.parentGroup == item.parentGroup)
       switch accessGroup {
       | Some(val) => {
+          let updatedScopes = switch val.scopes {
+          | ["write", "read"] => ["read", "write"]
+          | [_, _] => val.scopes
+          | _ => val.scopes
+          }
```

Comment:
Can we directly do Array.sort in this case?

Reply:
Array.sort returns unit and records in rescript are immutable so that's why sorting was not working in this case.

Reply:
do sorting at the line no 78 itself

Reply:
scopes is expecting type array<string> if i do sorting at line 78 it'll cause type error